### ISSUE TITLE
backend: create credential at Lissi API

### ISSUE THREAD

**Issue by veitmo** [amosproj/amos2022ss04-digital-identity#111]
## User story

1. As a {HR-Employee}
2. I want / need {to create a new credential which I can later assign to a DI}
3. So that {I can create a Mitarbeiterausweis for example}

## Acceptance criteria
* backend sends request to Lissi API to create a credential with data (which is later received from frontend)
* response from Lissi API is successfully received 

## Definition of done (DoD)

- Added only after week 5
- The same for all features
- Here goes the project specific part

## DoD general criteria

- Feature has been fully implemented
- Feature has been merged into the mainline
- All acceptance criteria were met
- Product owner approved features
- All tests are passing
- Developers agreed to release
